File: src/me/wikyfg/ane/commands/Rain.java
package me.wikyfg.ane.commands;

import me.wikyfg.ane.ANEMain;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class Rain implements CommandExecutor {

    private ANEMain main;

    public Rain(ANEMain main) {
        this.main = main;
    }

    @Override
    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
        Player p = (Player) sender;
        if (args.length > 0) {
            p.sendMessage(ChatColor.RED + "Te has confundido de argumentos, prueba a poner /rain.");
            return false;
        }
        p.getWorld().setStorm(true);
        p.getWorld().setWeatherDuration(20000);
        return false;
    }
}

File: src/me/wikyfg/ane/commands/Rain.kt
package me.wikyfg.ane.commands

import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class Rain : CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, label: String, args: Array<String>): Boolean {
        val p = sender as? Player
        p?.apply {
            if (!isOp) {
                sendMessage("${ChatColor.RED}Lo siento, no tienes permisos para hacer esto.")
                return true
            }
            if (args.isNotEmpty()) {
                sendMessage("${ChatColor.RED}Te has confundido de argumentos, prueba a poner /rain.")
                return true
            }
            world.setStorm(true)
            sendMessage("${ChatColor.GOLD}Clima cambiado a lluvia.")
            world.weatherDuration = 20000
        }
        return true
    }
}